app/Database/seeds/settings.yml

```diff
@@ -235,6 +235,27 @@
   options: ''
   type: number
   description: 'How much the load factor can vary per-flight'
+- key: flights.different_cargo_load_factor
```

Comment:
Suggesting to use `flights.use_cargo_load_factor` instead `different`

Reply:
I agree